homeassistant/components/fritz/sensor.py

```diff
@@ -100,7 +101,7 @@ def name(self):
         return self._name
 
     @property
-    def device_class(self) -> str | None:
+    def device_class(self) -> Any:
```

Comment:
The type for the base class is `str | None` why would you change this to `Any`?

Reply:
Fixed